### ISSUE TITLE
Questions with restricted access can be answered any time

### ISSUE THREAD

**Issue by olunczek** [University-of-Potsdam-MM/Reflect.UP#148]
The restriction of Feedback Questions does not work correctly: Questions can be answered as soon as  created, before and after the actual accessibility time.

**Comment by alekiy** [comment]
see https://github.com/University-of-Potsdam-MM/reflect-local_reflect/commit/de853cf8a7531d4c2b52d1a680b8429645aee4c5

**Comment by olunczek** [comment]
Works fine now.
